components/Projects.jsx

```diff
@@ -1,6 +1,13 @@
 export default () => (
   <div>
     <h2>Our Projects</h2>
+    <h3>InsightAI</h3>
+    <p>
+      Deep learning live object and depth recognition with binaural AR audio
+      spatialization and text-to-speech to help the visually impaired see the
+      world with sound using only a smartphone and headphones.
+    </p>
+    <p>Top 10 and 1st Place for Best Hack that Helps the Community</p>
```

Comment:
Top 10 out of 240 college-level teams and 1st Place for Best Hack that Helps the Community presented by CBRE